OrbitGl/EventTrack.cpp

```diff
@@ -21,21 +22,23 @@ std::string EventTrack::GetTooltip() const {
 }
 
 //-----------------------------------------------------------------------------
-void EventTrack::Draw(GlCanvas* canvas, bool picking) {
+void EventTrack::Draw(GlCanvas* canvas, PickingMode picking_mode) {
   Batcher* batcher = canvas->GetBatcher();
   PickingManager& picking_manager = canvas->GetPickingManager();
 
-  constexpr float kNormalZ = -0.1f;
-  constexpr float kPickingZ = 0.1f;
-  float z = kNormalZ;
+  const bool picking = picking_mode != PickingMode::kNone;
+  // The sample indicators are at z == 0 and do not respond to clicks, but
+  // have a tooltip. For picking, we want to draw the event bar over them if
+  // handling a click, and underneath otherwise.
+  // This simulates "click-through" behavior.
+  const float eventBarZ = picking_mode == PickingMode::kClick ? 0.1f : -0.1f;
```

Comment:
There are all these different z value definitions in GlCanvas, like GlCanvas::Z_VALUE_EVENT. Would it make sense to add them there?

Reply:
Good point, I moved them there.